tests/test_decorators.py

```diff
@@ -20,7 +20,13 @@
 
 import unittest
 import time
-from io import BytesIO
+
+try:
+	from cStringIO import StringIO
+except ImportError:
+	# Python 3
+	from io import StringIO
```

Comment:
The indentation here is doubled (did you use a tab character by accident)?

Reply:
Yes, more by habit than accident...